src/public/parts/shells/social/public/index.html

```
@@ -282,17 +282,17 @@ <h2 class="view-title" data-i18n="social.search.viewTitle"></h2>
 					</div>
 					<div class="search-filters">
 						<input type="text" id="searchViewAuthor" class="input input-bordered input-sm" data-i18n="social.search.filterAuthor" placeholder="" />
-						<select id="searchViewMedia" class="select select-bordered select-sm">
+						<select id="searchViewMedia" class="select select-bordered select-sm" data-i18n="social.search.filterMedia">
 							<option value="" data-i18n="social.search.filterMediaAll"></option>
 							<option value="image" data-i18n="social.search.filterMediaImage"></option>
 							<option value="video" data-i18n="social.search.filterMediaVideo"></option>
 						</select>
 						<input type="text" id="searchViewTag" class="input input-bordered input-sm" data-i18n="social.search.filterTag" placeholder="" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)